src/libraries/Microsoft.Extensions.DependencyInjection/src/Microsoft.Extensions.DependencyInjection.csproj

```diff
@@ -15,23 +15,32 @@
     <ILEmitBackend Condition="'$(TargetFramework)' != 'netstandard2.0'">True</ILEmitBackend>
     <DefineConstants Condition="'$(ILEmitBackend)' == 'True'">$(DefineConstants);IL_EMIT</DefineConstants>
     <DefineConstants Condition="$(TargetFramework.StartsWith('net4')) and '$(ILEmitBackendSaveAssemblies)' == 'True'">$(DefineConstants);SAVE_ASSEMBLIES</DefineConstants>  
+    <DisableImplicitAssemblyReferences>false</DisableImplicitAssemblyReferences>
```

Comment:
(nit) I don't think this needs to be in this property group because it isn't a "TargetFramework Derived Property". I can just be in the above PropertyGroup, under "PackageDescription".